components/org.wso2.finance.open.banking.conformance.mgt/src/main/java/org/wso2/finance/open/banking/conformance/mgt/db/SQLConstants.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * This class will hold all SQL queries needed for the
+ * application as constants.
+ */
+package org.wso2.finance.open.banking.conformance.mgt.db;
+
+public class SQLConstants {
+    /* Create Tables */
```

Comment:
add a private constructor to hide the implicit public one

Reply:
done